spec/meal_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'meal'
+  describe Meal do 
+    let (:meal) {Meal.new 'pasta', 7}
+    it 'has a name when initialized'do 
+      expect(meal.name).to eq 'pasta'
+    end
+    it 'has a price when initialized' do
+      expect(meal.price).to eq 7
+    end
+
+end
```

Comment:
Lint/BlockAlignment: `end` at 11, 0 is not aligned with `describe Meal do ` at 2, 2.